Build/glass/cache_glass.yml

```diff
@@ -0,0 +1,41 @@
+schedules:
+  - cron: "0 0 * * 0" # Runs at 00:00 UTC every Sunday
+    displayName: Weekly build
+    branches:
+      include:
+        - main
+    always: true
+
+steps:
+  - task: AzureCLI@2
+    displayName: 'Acquire an AAD token from a User-Assigned Managed Identity and save it as secret variable DropToken'
+    inputs:
+      azureSubscription: 'PylanceSecureVsIdePublishWithManagedIdentity'
+      scriptType: 'pscore'
+      scriptLocation: 'inlineScript'
+      inlineScript: |
+        # if this fails, check out this bash script that includes diagnostics:
+        # https://gist.github.com/johnterickson/19f80a3e969e39f1000d118739176e62
+        
+        # Note that the resoruce is specified to limit the token to Azure DevOps
+        $token = az account get-access-token --query accessToken --resource 499b84ac-1321-427f-aa17-267ca6975798 -o tsv
+        Write-Host "##vso[task.setvariable variable=DropToken;issecret=true]$token"      
+
+  # Setup the glass test folder
+  - task: PythonScript@0
+    displayName: 'Setup glass test folder'
+    env:
+      SYSTEM_ACCESSTOKEN: $(DropToken)
+    inputs:
+      scriptSource: 'filePath' # 'filePath' | 'inline'. Required. Script source. Default: filePath.
+      scriptPath: $(Build.SourcesDirectory)/Build/setup_glass.py
+      arguments: '--authTokenVariable SYSTEM_ACCESSTOKEN --buildOutput $(Build.BinariesDirectory)\raw\binaries'
```

Comment:
This is going to copy the build output from PTVS into this folder too? Maybe setup-glass should be modified to be able to skip that step?

Reply:
To be fair, this should work regardless because the run_glass.py file also recopies the build output